meson.build

```diff
@@ -23,7 +23,7 @@ host_system = host_machine.system()
 cc = meson.get_compiler('c')
 
 cc_warnings = []
-cc_features = []
+cc_features = ['-fno-math-errno']
 
 if cc.get_id() != 'msvc'
     cc_features += '-D_POSIX_C_SOURCE=200809L'
```

Comment:
We know this will never work for MSVC, so we can skip this just like the POSIX define

Reply:
Ok, moved into the `msvc` branch.